contrib/bitcoin-cli.bash-completion

```diff
@@ -1,141 +0,0 @@
-# bash programmable completion for bitcoin-cli(1)
```

Comment:
Why?

I'd argue the file should stay in the old directory. All the functional test needs is read-access to the file. You can get the file path in the test with:

```py
os.path.join(config["environment"]["SRCDIR"], 'contrib', ...

Reply:
Thanks, I was missing the `config["environment"]` variable. Branch updated.